### DISCUSSION TITLE
Memory usage includes Cached usage

### DISCUSSION THREAD

**Discussion by IamTheCarl** [timmo001/system-bridge#768] [General]
### Description

This may not be a bug depending on your perspective.
On Linux the memory usage report is including the cached memory as well.

This is much higher than tools like system monitor or htop will report, which will confuse most users.

Output of `uname -a` on my system.
```
Linux thecarl-Lenovo 5.4.0-80-generic #90-Ubuntu SMP Fri Jul 9 22:49:44 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

### What Platform / OS are you running?

Linux

### What version are you running?

v2.5.0

### Anything in the logs or a references that might be useful?

```txt
My log file is repeating the following message quite frequently:

2021-08-27 22:36:33 warn: undefined
```

But I think this is unrelated.
```


### Additional information

I'm not familiar with typescript but I do know how to get the normal memory usage in C.

```c
#include <sys/resource.h>

rusage usage;
getrusage(RUSAGE_SELF, &usage);

// The memory usage will be given to you in kilobytes, so multiply by 1000 to get bytes.
memoryUsage = usage.ru_maxrss * 1000;
```

Perhaps the terms will be similar in the library you are using.

Also here's the output of `uname -a` in case you need more specifics on my OS.
```
Linux carl-Lenovo 5.4.0-80-generic #90-Ubuntu SMP Fri Jul 9 22:49:44 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by timmo001** [comment]
All memory stats are pulled from this library:

https://systeminformation.io/memory.html

https://github.com/sebhildebrandt/systeminformation/blob/master/lib/memory.js

How this works might be something you want to look into to see what data you are going for

**Reply by IamTheCarl** [reply]
Ah, that's very helpful.

The active memory is what you want. It only inculdes memory that is actively allocated by an application.
http://books.gigatux.nl/mirror/linuxperformanceguide/0131486829/ch03lev1sec2.html

Also a quick off topic question: I'm new to discussions. Why was it preferred over an issue? I just want to make sure my etiquette is good.
